app/javascript/app/components/country/country-ndc-overview/country-ndc-overview-component.jsx

```diff
@@ -53,13 +57,22 @@ class CountryNdcOverview extends PureComponent {
 
   renderCards() {
     const { sectors, values } = this.props;
+    const renderSubtitle = text => <h4 className={styles.subTitle}>{text}</h4>;
+    const adaptationTitle = (
+      <h4 className={styles.subTitle}>Adaptation contribution</h4>
+    );
     return (
       <div>
```

Comment:
This `div` is also a column container.  I think is better to add a `class` to identify it as such to avoid eventual deletions in the future.